linera-views/src/backends/rocks_db.rs

```diff
@@ -450,6 +448,17 @@ impl AdminKeyValueStore for RocksDbStoreInternal {
         })
     }
 
+    async fn new_benchmark_config() -> Result<RocksDbStoreConfig, RocksDbStoreInternalError> {
+        let path_with_guard = create_rocks_db_test_path();
+        let common_config = create_rocks_db_common_config();
+        let spawn_mode = RocksDbSpawnMode::BlockInPlace;
```

Comment:
Is that the only case where it's different?? Why not using the formula then?

Reply:
Well, for the previous PR, which introduced the spawn_mode, we initially had it set up from `get_spawn_mode_from_runtime`(well not really, the function was not yet introduced) directly in the constructor of the store. However, this was viewed as a problem since it can indeed lead to hard to resolve bugs. I think it is better to have a deterministic code.